lib/nextAuthAdapter.ts

```diff
@@ -46,18 +44,7 @@ export default function Adapter() {
       if (!user.id) {
         return null;
       }
-      const email = user.id;
-      // ?? we already do the validation in signIn callback (see pages/api/auth/[...nextauth].ts)
-      if (validateEmailWithACL(email)) {
-        return {
-          id: email,
-          name: email.split('@')[0],
-          email,
-          role: 'admin',
-          emailVerified: new Date(),
-        } as AdapterUser;
-      }
-      return null;
+      return user;
```

Comment:
@deepakprabhakara  The above change in `updateUser` results in the user object in the session being empty (thereby the top right user avatar does not display anything). The `user` turns out to be just `{ id: 'aswin@boxyhq.com', emailVerified: 2023-01-10T17:56:31.027Z }`. We have to extract the profile similar to `getUserByEmail`